near-sdk/src/lib.rs

```diff
@@ -453,6 +453,45 @@ extern crate quickcheck;
 /// - [examples/adder](https://github.com/near/near-sdk-rs/blob/9596835369467cac6198e8de9a4b72a38deee4a5/examples/adder/src/lib.rs?plain=1#L31)
 /// - [examples/callback-results](https://github.com/near/near-sdk-rs/blob/9596835369467cac6198e8de9a4b72a38deee4a5/examples/callback-results/src/lib.rs?plain=1#L51)
 ///
+/// ### Implementation details of `#[callback_unwrap]` macro and **host functions** calls used
+///
+/// ```rust
+/// # use near_sdk::near;
+/// # #[near(contract_state)]
+/// # pub struct Contract { /* .. */ }
+/// #[near]
+/// impl Contract {
+///     pub fn method(
+///         &mut self,
+///         regular: String,
+///         #[callback_unwrap] one: String,
+///         #[callback_unwrap] two: String
+///     ) { /* .. */ }
+/// }
+/// ```
+///
+/// For above `method` using the attribute on arguments, changes the body of function generated in  [`#[near]` on mutating method](near#for-above-mutating-method-near-macro-defines-the-following-function)
```

Comment:
this pr depends on #1307, as the doc anchor is first added there. 
the broken link is not detected by 
```yaml
        RUSTDOCFLAGS: -D warnings
      run: |
        cargo doc -p near-sdk --features unstable,legacy,unit-testing,__macro-docs,__abi-generate
```

https://github.com/rust-lang/rust/issues/137740